Update InAppNotification style

ismaelestalayo opened the pull request:
<!-- 🚨 Please Do Not skip any instructions and information mentioned below as they are all required and essential to evaluate and test the PR. By fulfilling all the required information you will be able to reduce the volume of questions and most likely help merge the PR faster 🚨 -->

<!-- 📝 It is preferred if you keep the "☑️ Allow edits by maintainers" checked in the Pull Request Template as it increases collaboration with the Toolkit maintainers by permitting commits to your PR branch (only) created from your fork.  This can let us quickly make fixes for minor typos or forgotten StyleCop issues during review without needing to wait on you doing extra work. Let us help you help us! 🎉 -->


## Fixes #
<!-- Add the relevant issue number after the "#" mentioned above (for ex: Fixes #1234) which will automatically close the issue once the PR is merged. -->

Fixes #3770 by bringing the style of the `InfoBar` to the `InAppNotification` control.

<!-- Add a brief overview here of the feature/bug & fix. -->

## PR Type
What kind of change does this PR introduce?
<!-- Please uncomment one or more that apply to this PR. -->

<!-- - Bugfix -->
- Feature
<!-- - Code style update (formatting) -->
<!-- - Refactoring (no functional changes, no api changes) -->
<!-- - Build or CI related changes -->
<!-- - Documentation content changes -->
<!-- - Sample app changes -->
<!-- - Other... Please describe: -->


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

#3770



## What is the new behavior?
<!-- Describe how was this issue resolved or changed? -->

The coloring, font size and margins were updated to match the new `InfoBar` control.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../readme.md#supported)
- [ ] Pull Request has been submitted to the documentation repository [instructions](..\contributing.md#docs). Link: <!-- docs PR link -->
- [ ] Sample in sample app has been added / updated (for bug fixes / features)
    - [ ] Icon has been created (if new sample) following the [Thumbnail Style Guide and templates](https://github.com/windows-toolkit/WindowsCommunityToolkit-design-assets)
- [ ] New major technical changes in the toolkit have or will be added to the [Wiki](https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki) e.g. build changes, source generators, testing infrastructure, sample creation changes, etc...
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Header has been added to all new source files (run *build/UpdateHeaders.bat*)
- [x] Contains **NO** breaking changes

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. 
     Please note that breaking changes are likely to be rejected within minor release cycles or held until major versions. -->

Kyaa-dost commented:
@ismaelestalayo Thanks for submitting the PR. Can you please sign the CLA requirement above? ⬆️

ismaelestalayo commented:
> @ismaelestalayo Thanks for submitting the PR. Can you please sign the CLA requirement above? ⬆️

I already signed it, but the bot sent the CLA message twice, and only the first one updated to show that I signed it.
Clicking the link again just shows my signed agreement, which is not editable

Kyaa-dost commented:
Looks good now. Thanks, @ismaelestalayo for double checking.

michael-hawker commented:
@msftbot only merge if @vgromfeld approves.

@vgromfeld which style are you using currently? Let me know how this looks to you.

vgromfeld commented:
We are currently using a custom template to tweak some parts for our apps (mostly alignments) but with this change, it seems that we will not longer need them.

michael-hawker reviewed (changes requested):
@ismaelestalayo not sure if this is related to this PR or a regression from 6.1, but looks like the 'without DataTemplate' example doesn't position the close button properly:

![image](https://user-images.githubusercontent.com/24302614/108769603-11968080-750e-11eb-9639-d6465a89abb5.png)

Compared to current 6.1 sample app:

![image](https://user-images.githubusercontent.com/24302614/108769637-1e1ad900-750e-11eb-871b-d8d65ac767f4.png)

Is this something you'd mind taking a quick look at to fix in either case?

----

Also, to clarify, assuming some of the others have moved to the top-right based on the InfoBar template?

![image](https://user-images.githubusercontent.com/24302614/108769766-49052d00-750e-11eb-96fc-bc2bdbac7bef.png)

vs. 

![image](https://user-images.githubusercontent.com/24302614/108769785-502c3b00-750e-11eb-976f-3bb19d71e28e.png)

Just wanted to make sure it was intentional.

Thanks!

ismaelestalayo commented:
> @ismaelestalayo not sure if this is related to this PR or a regression from 6.1, but looks like the 'without DataTemplate' example doesn't position the close button properly:
> 
> ![image](https://user-images.githubusercontent.com/24302614/108769603-11968080-750e-11eb-9639-d6465a89abb5.png)

Fixed in the next code i'm gonna push:

![image](https://user-images.githubusercontent.com/17100634/108871698-b0e47200-75f9-11eb-9eb3-b7a593f19ffc.png)


I also fixed the "notification with Drop Shadow" having the X in the center, as my intention was to put it in the top right corner like `InfoBar`, yes:

![image](https://user-images.githubusercontent.com/17100634/108871665-a9bd6400-75f9-11eb-9fb3-31c0c13d670e.png)

ismaelestalayo commented:
Oh, you can see in the changes I made, but I took this oportunity to make the text and buttons from the examples smaller (font from 16 to 14, like the default, and buttons from 40x120 to 30x100)

michael-hawker commented:
@ismaelestalayo I believe the default is 32, no? As it's a multiple of 4. Currently at 150% DPI the bottom of the buttons don't scale properly:

![image](https://user-images.githubusercontent.com/24302614/108899144-efb00300-75cc-11eb-805f-73b8517ab2f4.png)

michael-hawker commented:
I've fixed locally, I'll see if I can just push the small change to 32:

![image](https://user-images.githubusercontent.com/24302614/108900601-a791e000-75ce-11eb-81b5-b54ed8429148.png)

michael-hawker commented:
Oh, I just noticed this is based off the master of your fork. It needs to be in a feature branch. I'll rebase this off my fork and just push and commit, since we're ready to go. I'll close this PR for now and ping you on the new one.
